api/v1alpha1/oidc_types.go

```diff
@@ -75,3 +81,17 @@ type OIDCProvider struct {
 	// +optional
 	TokenEndpoint *string `json:"tokenEndpoint,omitempty"`
 }
+
+// OIDCCookieNames defines the names of cookies to use in the Envoy OIDC filter.
+type OIDCCookieNames struct {
+	// The name of the cookie used to store the BearerToken in the
+	// [Authentication Request](https://openid.net/specs/openid-connect-core-1_0.html#AuthRequest).
+	// If not specified, defaults to "BearerToken-(randomly generated uid)"
+	// +optional
+	BearerToken *string `json:"bearerToken,omitempty"`
```

Comment:
The name `BearerToken` is confusing in this context.  Let's rename it `AccessToken` to keep consistent with https://github.com/envoyproxy/gateway/pull/3423

```suggestion
	AccessToken *string `json:"accessToken,omitempty"`
```